Minor updates to Access Review Beta (#14738)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewrecommendationinsightsetting.md

---
title: "accessReviewRecommendationInsightSetting resource type"

> description: "In the Azure AD access reviews feature, the `accessreviewrecommendationinsightsetting` represents the settings associated with an insight included in the review."

Reviewer: ```suggestion
description: "In the Azure AD access reviews, the accessReviewRecommendationInsightSetting represents the settings associated with an insight provided for an access review."
```

author: "isabelleatmsft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewRecommendationInsightSetting resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

The **accessReviewRecommendationInsightSetting** allows you to configure insights for an [accessReviewScheduleDefinition](accessreviewscheduledefinition.md). Note that this is an abstract type with multiple derived types

## Relationships
None.

## JSON representation
The following is a JSON representation of the resource.
<!-- {

>   "blockType": "resource",
>   "@odata.type": "microsoft.graph.accessreviewrecommendationinsightsetting"

Reviewer: ```suggestion
  "blockType": "resource",
  "@odata.type": "microsoft.graph.accessReviewRecommendationInsightSetting"
```

}
-->
``` json
{

>   "@odata.type": "#microsoft.graph.accessreviewrecommendationinsightsetting",

Reviewer: ```suggestion
  "@odata.type": "#microsoft.graph.accessReviewRecommendationInsightSetting",
```
